### ISSUE TITLE
Broadcasting error in `reset()` of `vec_env.py` running `hello_world.py`

### ISSUE THREAD

**Issue by DennisSoemers** [Farama-Foundation/MicroRTS-Py#102]
Going through the **Getting Started** steps (with minor modifications to get things to work as I detailed in Issue #101) , the `python hello_world.py` command terminates with the following error:

```
Traceback (most recent call last):
  File "/mnt/c/Apps/MicroRTS-Py/hello_world.py", line 38, in <module>
    envs.reset()
  File "/mnt/c/Apps/MicroRTS-Py/gym_microrts/envs/vec_env.py", line 188, in reset
    obs = [self._encode_obs(np.array(ro)) for ro in responses.observation]
  File "/mnt/c/Apps/MicroRTS-Py/gym_microrts/envs/vec_env.py", line 188, in <listcomp>
    obs = [self._encode_obs(np.array(ro)) for ro in responses.observation]
  File "/mnt/c/Apps/MicroRTS-Py/gym_microrts/envs/vec_env.py", line 193, in _encode_obs
    obs = obs.reshape(len(obs), -1).clip(0, np.array([self.num_planes]).T - 1)
  File "/home/dennis/.cache/pypoetry/virtualenvs/gym-microrts-VkXpJVXw-py3.9/lib/python3.9/site-packages/numpy/core/_methods.py", line 159, in _clip
    return _clip_dep_invoke_with_casting(
  File "/home/dennis/.cache/pypoetry/virtualenvs/gym-microrts-VkXpJVXw-py3.9/lib/python3.9/site-packages/numpy/core/_methods.py", line 113, in _clip_dep_invoke_with_casting
    return ufunc(*args, out=out, **kwargs)
ValueError: operands could not be broadcast together with shapes (6,256) () (5,1)
```

**Comment by DennisSoemers** [comment]
Yep, I did, and I'm inside the venv it created (see how there's a couple of `/home/dennis/.cache/pypoetry/virtualenvs/`-like filepaths in the trace, I assume the `pypoetry` in there means it's stuff that got installed through poetry).

Python version: 3.9.16
`numpy.__version__`: 1.21.4

**Comment by DennisSoemers** [comment]
Oh, I do just realise that I probably didn't re-run `poetry install` after switching from the fork of the Farama Foundation repo to the fork of your fork (re: the other Issue I posted). But I'm probably not gonna be able to test this again until Thursday. So, please don't waste any more time thinking about this issue until I can test and report back.

**Comment by vwxyzjn** [comment]
I think it's probably you cloned `gym-microrts` instead of the new `Microrts-Py`. I have updated the docs in #103. Meanwhile https://asciinema.org/a/586754 shows my installation end-to-end

**Comment by DennisSoemers** [comment]
The problem ended up being something related to the cloning of submodules. Instead of working with the original repos, I wanted to work with my own forks of both MicroRTS-Py and MicroRTS (which so far do not have any changes in comparison to their original repos, but I may want to add changes in the future).

So, I did the pulling of `https://github.com/DennisSoemers/MicroRTS-Py` without `--recursive`, and then cloned the two submodules in their appropriate locations manually (where for `MicroRTS` I again had my own fork, and for `static-files` just took yours). Apparently, this leads to the above error for some reason. 

Now, instead, I just changed the `.gitmodules` file in my own fork of `MicroRTS-Py` to pull in my `MicroRTS` fork (see https://github.com/DennisSoemers/MicroRTS-Py/commit/6941ab75ca96fda88043ceb1fcc4327265a468ea), which lets me pull the outer repo with `--recursive` and gives me both of my forks. This is going to be annoying if, in the future, I ever want to send any Pull Requests... but for now it works.

I won't close the issue, in case the above text is interesting for you / points to anything that warrants changing/fixing in your opinion, but for me personally the issue is resolved, so I'm also happy if you want to close the issue.
